ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -162,13 +175,48 @@ jobs:
           body: results/body
           subject: results/subject
     ensure:
-      aggregate:
-      - put: geode-build-artifact
-        params:
-          file: built-geode/geodefiles-*.tgz
+      do:
+      - task: rsync_code_down
+        {{ alpine_tools_config()|indent(8) }}
+          run:
+            path: geode-ci/ci/scripts/rsync_code_down.sh
+          inputs:
+          - name: geode-ci
+          - name: instance-data
+          outputs:
+          - name: geode-results
+        timeout: 5m
       - put: geode-build-version
         params:
           file: results/number
+      ensure:
+        do:
+        - aggregate:
+          - task: archive_results
+            {{ alpine_tools_config()|indent(12) }}
+              params:
+                ARTIFACT_SLUG: {{build_test.ARTIFACT_SLUG}}
+                GRADLE_TASK: {{build_test.GRADLE_TASK}}
+                MAINTENANCE_VERSION: {{repository.branch}}
+                PUBLIC_BUCKET: ((!public-bucket))
+                SERVICE_ACCOUNT: ((!concourse-gcp-account))
+                TAR_GEODE_BUILD_ARTIFACTS: true
+              run:
+                path: geode-ci/ci/scripts/archive_results.sh
+              inputs:
+              - name: concourse-metadata-resource
+              - name: geode-ci
+              - name: geode-build-version
+              - name: geode-results
+            timeout: 1h
+          - task: stop_instance
+            {{ alpine_tools_config()|indent(12) }}
```

Comment:
Indentation seems off here.

Reply:
It is not?  `apline_tools_config()` should be indented one under other `- task` declarations / inline with the `task` keyword.  Both visually for the pre-Jinja-interpolation and the `indent(12)` for after the Jinja interpolation should be correct.